portal-web/test/functional/com/liferay/portalweb/tests/enduser/contentdashboard/ContentDashboard.testcase

```diff
@@ -1788,4 +1788,109 @@ definition {
 			title = "WC Title");
 	}
 
+	@description = "LPS-116987 When the user makes a change on web content and creates a draft version, the Info panel shows the version for the last version approved and the version of the draft "
+	@priority = "5"
+	test ApprovedAndDraftStatus {
+		property portal.acceptance = "true";
+      
+	    task ("Create a Web Content and set a Display Page") {
+			JSONWebcontent.addWebContent(
+				content = "Webcontent Content",
+				groupName = "Test Site Name",
+				title = "WC Content");
+		}
+        task ("Navigate to the content Dashboard, click edit web content and save as draft") {
+			ApplicationsMenu.gotoPortlet(
+				category = "Content",
+				panel = "Applications",
+				portlet = "Content Dashboard");
+
+			Click(locator1 = "ContentDashboard#WEB_CONTENT_MENU_DROPDOWN_BUTTON");
+
+			Click(locator1 = "ContentDashboard#EDIT_OPTION");
+
+			CKEditor.addContentInWebContent(
+				content = "Web Content Content",
+				fieldLabel = "content");
+
+			Click(locator1 = "Button#SAVE_AS_DRAFT");
+			
+			Click(locator1 = "Icon#BACK");
+		}
```

Comment:
I think all these Click lines should be in a macro, like saveAsDraft(); or something like that